addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -197,11 +197,13 @@ const DynamicSnippet = publicWidget.Widget.extend({
         this.trigger_up('widgets_stop_request', {
             $target: $templateArea,
         });
-        const mainPageUrl = this._getMainPageUrl();
```

Comment:
It's not related to the fix but I moved `this._getMainPageUrl();` inside the `if` because we don't need to run it if the link doesn't need to be set.